app/preact/package.json

```diff
@@ -35,7 +35,7 @@
     "@babel/plugin-transform-react-jsx": "^7.3.0",
     "@storybook/addons": "5.3.0-alpha.2",
     "@storybook/core": "5.3.0-alpha.2",
-    "common-tags": "^1.8.0",
```

Comment:
Did you intent to remove the dependency on `common-tags`

Reply:
whoops! all fixed now